src/main/java/mate/academy/model/User.java

```diff
@@ -1,8 +1,8 @@
 package mate.academy.model;
 
 public class User {
-    private String email;
-    private String password;
+    private final String email;
+    private final String password;
```

Comment:
any reasons to do that?
```suggestion
    private String email;
    private String password;
```